test/command/suite/load/time/cast/string/iso8601/normal.test

```diff
@@ -0,0 +1,20 @@
+table_create Logs TABLE_NO_KEY
+column_create Logs created_at COLUMN_SCALAR Time
+
+load --table Logs
+[
+{"created_at": "2000-01-01T10:00:00Z"},
+{"created_at": "2000-01-01t10:00:00z"},
+{"created_at": "2000-01-01T10:00:00+00:00"},
+{"created_at": "2000-01-01T11:01:00+01:01"},
+{"created_at": "2000-01-01T21:11:00+11:11"},
+{"created_at": "2000-01-01T08:59:00-01:01"},
+{"created_at": "1999-12-31T22:49:00-11:11"},
+{"created_at": "2000-01-01T11:00:00+01"},
+{"created_at": "2000-01-01T09:00:00-01"},
+{"created_at": "2000-01-01T11:01:00.123+01:01"},
+{"created_at": "2000-01-02T09:00:00+23:00"},
+{"created_at": "2000-01-01T10:59:00+00:59"},
+{"created_at": "2000/01/01T10:00:00Z"}
+]
+select Logs --limit -1
```

Comment:
How about showing also input text and formatting time value as ISO format string? It improves debugability.

```text
plugin_register functions/time

table_create Logs TABLE_NO_KEY
column_create Logs created_at COLUMN_SCALAR Time
column_create Logs created_at_text COLUMN_SCALAR ShortText

load --table Logs
[
{"created_at":      "2000-01-01T10:00:00Z",
 "created_at_text": "2000-01-01T10:00:00Z"},
...
]

select Logs --limit -1 --output_columns 'time_format_iso8601(created_at), created_at_text`

Reply:
How about `valid.test` and `invalid.test` for file names?

Reply:
I have fixed it.